docs/source/API/core/view/realloc.md

```diff
@@ -9,6 +9,7 @@ Usage:
   ```
 
 Reallocates a view to have the new dimensions. Can grow or shrink, and will not preserve content.
+May not modify the view, if sizes already match.
```

Comment:
Maybe a native speaker can chime in but I feel like "may not" is unfortunate because it could be interpreted as "[presumably the implementation] is not allowed"

Reply:
Yeah, that is a reasonable mis-interpretation. The intended meaning would be conveyed (with more explicit precision) by "Might not modify the view, if its extents match those requested"

Reply:
@PhilMiller , @dalg24 - regarding line 12 above; why not be definitive, and say "if sizes already match, the View will not be modified."  Here, "may" suggests other possible outcomes, and there are none, if the requested sizes are the same as the current sizes , correct?

Reply:
This is reaching back to very old discussion, but I believe the intention was to avoid committing the implementation to any particular outcome. Once it's documented to definitely do something, that's a nearly-eternal promise.